docs/openjsf/application.md

```diff
@@ -0,0 +1,193 @@
+
+We, the JSON Schema collaborators, also believe this.
+
+We feel the approach laid out in the CPC charter is reasonable.
+
+We identify with the need for collaboration between JavaScript, web solutions, and related technologies.
+
+
+## Impact and users of the project
+> What is the ecosystem impact of the project?
+
+JSON Schema has far reaching impact in the JS ecosystem. It's used by libraries such as eslint and webpack.
+VSCode (and other IDEs) use JSON Schema to validate configuration files.
+GitHub reports roughly 9k dependent packages and over 9 million dependent repositories for the most popular implementation of JSON Schema (AJV).
+
+> Who are the biggest users of the project? (_if they are not JavaScript space, then will it ever expand to JavaScript space_)
+
+Libraries: eslint and webpack.
+Specifications: OpenAPI, AsyncAPI.
+Editors: Visual Studio Code, Eclipse IDE, Jetbrains IDEs.
+Organisations: Microsoft, Google, Amazon, Mozilla, SAP, IBM, HM Government, NIH, NASA, The Deutsche Bahn.
+
+Additionally, as of OpenAPI 3.1, all these organizations: https://www.openapis.org/membership/members
+
+> What is the potential long term impact and adoption?
+
+We are at a critical juncture of JSON Schema evolution.
+The extension mechanism, JSON Schema Vocabularies, allows for the standardised extension of JSON Schema to cater to non-validation use cases.
+
+Use cases such as code generation, form generation, database generation (and validation), have evolved, but are non-standard. We're already working with existing open source projects and major organisations to facilitate standardising these use cases.
+
+- How does it differ from other projects?
+
+JSON Schema is a domain-specific declarative language for validating and annotating JSON data.
+
+It cannot be compared to OpenAPI. OpenAPI 3.1 now defers the schema definitions to the most recent release of JSON Schema[1]. Nor can it be compared to JSON-LD, which is designed for linked data. Nor can it be compared to schema.org which provides data semantics.
+
+A possible comparison is to JSON Type Definition (RFC 8927), which is designed for code generation.
+By way of comparison, JSON Schema has an extensive test suite, many more implementations and languages supported, and has vastly more use in the ecosystem.
+
+Protocol Buffers and Avro do not just specify a structure, but define the "over the wire" format and serialisation method. JSON Schema is designed for validation of JSON data rather than data definitions, enabling the definition of more complex validation constraints than would otherwise be possible.
+
+In comparison to Joi, a popular and well used Javascript validation library, JSON Schema offers a language agnostic and portable validation mechanism.
+
+Many times we see comparisons made from one solution to JSON Schema. Often, they miss several key understandings.
```

Comment:
I don't think this line is necessary, especially if you aren't going to elaborate.